api/middleware/roleAuthentication.js

```diff
@@ -1,8 +1,9 @@
 const roleAuthentication = (...args) => (req, res, next) => {
   // role type is inside req.profile.body
   const { role } = req.profile;
+  console.log(role);
```

Comment:
Don't forget to remove the console.log

Reply:
I only see what Ryan said that the console log needs to be removed